<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-DI8EF55" failures="3" tests="8" name="E:/Rossy/Skillo/SKAU14TAF" time="189.363" errors="0" timestamp="2024-08-02T02:52:06 EEST">
  <testcase classname="com.skilo.gui.LoginTest" name="verifyAlreadyRegisteredUserCanSuccessfullyLogin" time="8.817"/>
  <testcase classname="com.skilo.gui.LoginTest" name="verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongPassword" time="49.703"/>
  <testcase classname="com.skilo.gui.LoginTest" name="verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithWrongUsername" time="49.268"/>
  <testcase classname="com.skilo.gui.LoginTest" name="verifyAlreadyRegisteredUserCanNotSuccessfullyLoginWithNotProvidedCredentials" time="48.841"/>
  <testcase classname="com.skilo.gui.PostTest" name="verifyUserCanCreateNewPost" time="0.004">
    <failure type="org.testng.internal.reflect.MethodMatcherException" message="
[public void com.skilo.gui.PostTest.verifyUserCanCreateNewPost(java.io.File,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserCanCreateNewPost([java.io.File arg0, java.lang.String arg1])
Arguments: [(java.lang.String) testingDemos,(java.lang.String) testing,(java.lang.String) testingDemos,(java.io.File) src\test\resources\uploads\testUpload.jpg,(java.lang.String) Testing create post caption]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.skilo.gui.PostTest.verifyUserCanCreateNewPost(java.io.File,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserCanCreateNewPost([java.io.File arg0, java.lang.String arg1])
Arguments: [(java.lang.String) testingDemos,(java.lang.String) testing,(java.lang.String) testingDemos,(java.io.File) src\test\resources\uploads\testUpload.jpg,(java.lang.String) Testing create post caption]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:915)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- verifyUserCanCreateNewPost -->
  <testcase classname="com.skilo.gui.PostTest" name="verifyUserCanCreateNewPostAndDeleteIt" time="0.001">
    <failure type="org.testng.internal.reflect.MethodMatcherException" message="
[public void com.skilo.gui.PostTest.verifyUserCanCreateNewPostAndDeleteIt(java.io.File,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserCanCreateNewPostAndDeleteIt([java.io.File arg0, java.lang.String arg1])
Arguments: [(java.lang.String) testingDemos,(java.lang.String) testing,(java.lang.String) testingDemos,(java.io.File) src\test\resources\uploads\testUpload.jpg,(java.lang.String) Testing create post caption]">
      <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void com.skilo.gui.PostTest.verifyUserCanCreateNewPostAndDeleteIt(java.io.File,java.lang.String)] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: verifyUserCanCreateNewPostAndDeleteIt([java.io.File arg0, java.lang.String arg1])
Arguments: [(java.lang.String) testingDemos,(java.lang.String) testing,(java.lang.String) testingDemos,(java.io.File) src\test\resources\uploads\testUpload.jpg,(java.lang.String) Testing create post caption]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:46)
at org.testng.internal.Parameters.injectParameters(Parameters.java:915)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- verifyUserCanCreateNewPostAndDeleteIt -->
  <testcase classname="com.skilo.gui.RegistrationTest" name="verifyUserCanRegisterWithValidData" time="7.815"/>
  <testcase classname="com.skilo.gui.RegistrationTest" name="verifyUserCanNotRegisterWithINValidData" time="6.95">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at org.testng.Assert.assertFalse(Assert.java:89)
at com.skilo.gui.RegistrationTest.verifyUserCanNotRegisterWithINValidData(RegistrationTest.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- verifyUserCanNotRegisterWithINValidData -->
  <testcase classname="com.skilo.gui.TestObject" name="@BeforeSuite setupTestSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupTestSuite -->
  <testcase classname="com.skilo.gui.TestObject" name="@AfterSuite cleanFiles" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite cleanFiles -->
  <testcase classname="com.skilo.gui.TestObject" name="@BeforeSuite setupTestSuite" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupTestSuite -->
  <testcase classname="com.skilo.gui.TestObject" name="@AfterSuite cleanFiles" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite cleanFiles -->
</testsuite> <!-- E:/Rossy/Skillo/SKAU14TAF -->
